schema_salad/makedoc.py

```diff
@@ -227,7 +234,7 @@ def typefmt(self,
                 nbsp=False,             # type: bool
                 jsonldPredicate=None    # type: Optional[Dict[str, str]]
                 ):
-        # type: (...) -> Union[str, unicode]
+        # type: (...) -> Union[str, Text]
```

Comment:
Separately mentioning is redundant in python3 indeed, but in case of python2 union states that this function can return ``ASCII`` strings as well as ``unicode`` strings. 
``# type: (...) -> Union[str, unicode]`` annotation was already present, I just made sure ``Text`` type makes it compatible with python3.